code/modules/research/designs/power_designs.dm

```diff
@@ -22,7 +22,7 @@
 	materials = list(MAT_METAL = 700, MAT_GLASS = 60)
 	construction_time=100
 	build_path = /obj/item/weapon/stock_parts/cell/high
-	category = list("Misc","Power Designs")
+	category = list("Misc","Power Designs","Machinery","hacked")
```

Comment:
Not sure about this one, hacking the autolathe being ridiculously easy, it might be too OP to let cargo make as many high-cap power cell as they want, especially to make stunprods.